test/sanity-check/api/contentType-test.js

```diff
@@ -0,0 +1,123 @@
+import path from 'path'
+import { expect } from 'chai'
+import { describe, it, setup } from 'mocha'
+import { jsonReader } from '../utility/fileOperations/readwrite.js'
+import { singlepageCT, multiPageCT, schema } from '../mock/content-type.js'
+import { contentstackClient } from '../utility/ContentstackClient.js'
+
+let client = {}
+let multiPageCTUid = ''
+let importCTUid = ''
+
+describe('Content Type api Test', () => {
+  setup(() => {
+    const user = jsonReader('loggedinuser.json')
+    client = contentstackClient(user.authtoken)
+  })
+
+  it('Create Single page ContentType Schema', done => {
```

Comment:
Change the message to "Should create a  single page ContentType Schema" 
Change it in all the test cases.

Reply:
will make that change change in next PR